anticipy/forecast_models.py

```diff
@@ -1494,6 +1500,22 @@ def f_dummy_list_date(a_x, a_date):
                 'date-like values: %s', list_check)
 
 
+def get_f_dummy_from_holiday(holiday):
```

Comment:
Unfortunately, the term Holiday is used by a class in pandas.tseries.calendar, so it would lead to confusion. I'm thinking something like:

```
get_model_from_date_list(name, *args):
    l_model_dummy = [get_model_dummy('model_dummy', l_date)
                     for l_date in args]
    f_model_prod = np.prod(l_model_dummy)
    f_model_sum = np.sum(l_model_dummy)

    def _f_init_params_date_list(
            a_x=None, a_y=None, a_date=None, is_mult=False):
        if is_mult:
            return np.ones(len(l_model_dummy))
        else:
            return np.zeros(len(l_model_dummy))

    def _f_model_date_list(a_x, a_date, params, is_mult=False, **kwargs):
        f_all_dummies = f_model_prod if is_mult else f_model_sum
        return f_all_dummies(a_x, a_date, params, is_mult, **kwargs)

    model_date_list = ForecastModel(
        name,
        len(l_model_dummy),
        _f_model_date_list,
        _f_init_params_date_list
    )
    return model_calendar  
```

Usage:
```
model_got=get_model_from_date_list('got_release',[date1, date2, date3])
model_multiple_params = get_model_from_date_list('multiple_date_events', [date1, date2],[date3,date4]
```